www/src/pages/components/chip/ios/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Chip
+description: Compact controls that allow for toggling and filtering
+---
+
+import { graphql } from 'gatsby';
+import { ComponentHeader } from 'components/thumbprint-components';
```

Comment:
I think you also need to add a line `<ComponentHeader data={props.data} />` right after this so that it actually shows the header (same in the Pill doc)